src/actions/authAction.js

```diff
@@ -20,7 +21,10 @@ export const registerUser = (userInfo, history) => async (dispatch) => {
         type: GET_USER_PROFILE,
         payload: res.data.user
       })
-      history.push('/');
+      toast.success(`Registration Succesful`)
+      setTimeout(() => {
+        activeForm('login')
+      }, 2000)
```

Comment:
@Kodecheff this looks good but can we also implement a notification to appear in the case of an unsuccessful registration attempt?

Reply:
@AuraOfDivinity I'm sorry about older commits. I was able to make a notification appear in case of registering an already existing user